[2024-03-12T09:53:57.741+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: two_separate_groups.l1_dynamic_generated_2 scheduled__2024-03-11T00:00:00+00:00 [queued]>
[2024-03-12T09:53:57.765+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: two_separate_groups.l1_dynamic_generated_2 scheduled__2024-03-11T00:00:00+00:00 [queued]>
[2024-03-12T09:53:57.766+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-12T09:53:57.808+0000] {taskinstance.py:2214} INFO - Executing <Task(BashOperator): l1_dynamic_generated_2> on 2024-03-11 00:00:00+00:00
[2024-03-12T09:53:57.821+0000] {standard_task_runner.py:60} INFO - Started process 893 to run task
[2024-03-12T09:53:57.835+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'two_separate_groups', 'l1_dynamic_generated_2', 'scheduled__2024-03-11T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/create_dag.py', '--cfg-path', '/tmp/tmpjhea_5w1']
[2024-03-12T09:53:57.841+0000] {standard_task_runner.py:88} INFO - Job 14: Subtask l1_dynamic_generated_2
[2024-03-12T09:53:58.063+0000] {task_command.py:423} INFO - Running <TaskInstance: two_separate_groups.l1_dynamic_generated_2 scheduled__2024-03-11T00:00:00+00:00 [running]> on host 4da48739a525
[2024-03-12T09:53:58.324+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='two_separate_groups' AIRFLOW_CTX_TASK_ID='l1_dynamic_generated_2' AIRFLOW_CTX_EXECUTION_DATE='2024-03-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-11T00:00:00+00:00'
[2024-03-12T09:53:58.326+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-12T09:53:58.328+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo ELLO WORLD']
[2024-03-12T09:53:58.343+0000] {subprocess.py:86} INFO - Output:
[2024-03-12T09:53:58.349+0000] {subprocess.py:93} INFO - ELLO WORLD
[2024-03-12T09:53:58.350+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-12T09:53:58.391+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=two_separate_groups, task_id=l1_dynamic_generated_2, execution_date=20240311T000000, start_date=20240312T095357, end_date=20240312T095358
[2024-03-12T09:53:58.463+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-12T09:53:58.485+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
